src/main/java/org/spongepowered/common/service/permission/SpongePermissionService.java

```diff
@@ -66,8 +67,8 @@
     private final SpongeSubjectCollection defaultCollection;
     private final SpongeSubject defaultData;
 
-    public SpongePermissionService(Game game) {
-        this.game = game;
+    public SpongePermissionService() {
```

Comment:
Would it be worth injecting the SpongeGame instance? Would require turning these into Singletons and creating them via the injector only if you need too.

Reply:
Yes, if we're okay with passing an injector to the service provider.

Reply:
I don't see an issue with it.